pom.xml

```diff
@@ -7,6 +7,95 @@
     <groupId>com.epam</groupId>
     <artifactId>EkaterinaDubinina</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
 
+    <properties>
+        <java.sourse.version>1.8</java.sourse.version>
+        <java.target.version>1.8</java.target.version>
+
+        <testng.version>7.1.0</testng.version>
+        <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+        <maven-compiler-plagin.version>3.8.1</maven-compiler-plagin.version>
+    </properties>
+
+    <profiles>
+        <profile>
+            <id>allTest</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <suite.xml>src/test/resources/hw1/AllTestSuite.xml</suite.xml>
+            </properties>
+
+        </profile>
+
+        <profile>
+            <id>sumSubtractTest</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <suite.xml>src/test/resources/hw1/SumSubtractTestSuite.xml</suite.xml>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>multiplyDevideTest</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <suite.xml>src/test/resources/hw1/MultiplyDevideTestSuite.xml</suite.xml>
+            </properties>
+        </profile>
+
+    </profiles>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>${testng.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.epam</groupId>
+            <artifactId>calculator</artifactId>
+            <version>1.0</version>
+            <scope>system</scope>
+            <systemPath>C:/Users/Катя/IdeaProjects/EkaterinaDubinina/src/main/resources/hw1/calculator-1.0.jar
```

Comment:
Why do you decide specify `absolute path` to the `jar` file? Are you sure that all team members have this path?

Reply:
The path for jar file was changed.